### PR TITLE
Related samples to ADO.NET documents, importing from .NET Framework-Part 4

### PR DESCRIPTION
**Pull request by DavoudEshtehari**
Related to the next part of porting documents from .NET Framework documents.

### CODE REVIEW

File: doc/samples/SqlConnection_OpenAsync_ContinueWith.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+using System;
```
Comment (David-Engel) [inline doc/samples/SqlConnection_OpenAsync_ContinueWith.cs:1]: I would leave out any samples that use the old Begin*/End* async pattern. We don't want to encourage any new use of those in MDS.

File: doc/samples/SqlClient_PerformanceCounter.cs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            // To avoid storing the connection string in your code,  
+            // you can retrieve it from a configuration file.  
+            return @"Data Source=.;Integrated Security=True;" +
+              "Initial Catalog=AdventureWorks";
+        }
+        private static string GetSqlConnectionString()
+        {
+            // To avoid storing the connection string in your code,  
+            // you can retrieve it from a configuration file.  
+            return @"Data Source=.;User Id=LowPriv;Password=Data!05;" +
```
Comment (David-Engel) [inline doc/samples/SqlClient_PerformanceCounter.cs:160]:
```suggestion
            return @"Data Source=.;User Id=<myUserID>;Password=<myPassword>;" +
```

File: doc/samples/SqlClient_PerformanceCounter.cs [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+              "Initial Catalog=AdventureWorks";
+        }
+
+        private static string GetSqlConnectionStringDifferent()
+        {
+            // To avoid storing the connection string in your code,  
+            // you can retrieve it from a configuration file.  
+            return @"Initial Catalog=AdventureWorks;Data Source=.;" +
+              "User Id=LowPriv;Password=Data!05;";
```
Comment (David-Engel) [inline doc/samples/SqlClient_PerformanceCounter.cs:169]:
```suggestion
              "User Id=<myUserID>;Password=<myPassword>;";
```

File: doc/samples/SqlClient_AsyncProgramming_SqlBulkCopy.cs [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+        // Replace the connection string if needed, for instance to connect to SQL Express: @"Server=(local)\SQLEXPRESS;Database=Demo;Integrated Security=true"
+        // static string connectionString = @"Server=(localdb)\V11.0;Database=Demo";
+        static string connectionString = @"Server=(local);Database=Demo;Integrated Security=true";
+
+        // static string marsConnectionString = @"Server=(localdb)\V11.0;Database=Demo;MultipleActiveResultSets=true;";
+        static string marsConnectionString = @"Server=(local);Database=Demo;MultipleActiveResultSets=true;Integrated Security=true";
+
+        // Replace the Server name with your actual sql azure server name and User ID/Password
+        static string azureConnectionString = @"Server=SqlAzure;User ID=myUserID;Password=myPassword;Database=Demo";
```
Comment (David-Engel) [inline doc/samples/SqlClient_AsyncProgramming_SqlBulkCopy.cs:34]:
```suggestion
        static string azureConnectionString = @"Server=SqlAzure;User ID=<myUserID>;Password=<myPassword>;Database=Demo";
```

File: doc/samples/SqlConnection_OpenAsync_ContinueWith.cs (hunk 1 above)
Comment (DavoudEshtehari) [reply doc/samples/SqlConnection_OpenAsync_ContinueWith.cs:1]: This sample relates to [this](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/asynchronous-programming#adding-the-new-asynchronous-feature-in-an-existing-application-mixing-old-and-new-patterns) section that I'm porting to ADO.Net documents.
Do you want me to remove that section too or just removing the samples is enough?

File: doc/samples/SqlClient_AsyncProgramming_ProviderModel.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
+using System;
+// <Snippet1>
+using System.Data.Common;
+using Microsoft.Data.SqlClient;
+using System.Threading.Tasks;
+
+class A
```
Comment (karinazhou) [inline doc/samples/SqlClient_AsyncProgramming_ProviderModel.cs:7]: Can we use `Program` here like other samples?

**Review by David-Engel — APPROVED** [review]

**Pull request merged**